modules/ssh/manifests/service.pp

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+class ssh::service {
+    case $::operatingsystem {
+        Darwin : {
+            exec {
+                # Using -w will enable the service for future boots, this
+                # command does tick the box for remote-login in the Sharing
+                # prefpane (survives reboot)
+                'turn-on-ssh' :
+                    command =>
+                    '/bin/launchctl load -w /System/Library/LaunchDaemons/ssh.plist',
+                    unless  =>
+                    "/usr/sbin/netstat -na | /usr/bin/grep -q 'tcp4.*\\*.22.*LISTEN'";
+            }
+
+            # Delete the com.apple.access_ssh group.  If present, this group limits
```

Comment:
We definitely shouldn't be deleting the ssh access group.  Instead, we should be adding them to the group.  I believe this already happens in the user module.